src/Evolution/Systems/GeneralizedHarmonic/Constraints.cpp

```diff
@@ -1325,6 +1326,60 @@ void constraint_energy(
     }
   }
 }
+
+template <typename DataType, size_t SpatialDim, typename Frame>
+Scalar<DataType> constraint_energy_normalization(
+    const tnsr::iaa<DataType, SpatialDim, Frame>& d_spacetime_metric,
+    const tnsr::iaa<DataType, SpatialDim, Frame>& d_pi,
+    const tnsr::ijaa<DataType, SpatialDim, Frame>& d_phi,
+    const tnsr::II<DataType, SpatialDim, Frame>& inverse_spatial_metric,
+    const Scalar<DataType>& sqrt_spatial_metric_determinant,
+    const double dimensional_constant) {
```

Comment:
At some point we'll need to add a function for computing the dimensional constant for evolutions where the total mass doesn't equal 1, but for now defaulting to 1.0 seems fine.